src/Server/StellarChat.Server.Api/Features/Idenitity/User/GetProfile/GetProfileHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace StellarChat.Server.Api.Features.Idenitity.User.GetProfile;
+
+internal class GetProfileHandler : ICommandHandler<GetProfile, GetProfileResponse>
+{
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
Consider using [IContext interface](https://github.com/ktutak1337/Stellar-Chat/blob/main/src/Shared/StellarChat.Shared.Abstractions/Contexts/IContext.cs) from the **Shared project** instead of **IHttpContextAccessor**. **IContext** is already designed to handle request information and the authenticated user, so it might be a more consistent approach across the codebase.